app/dashboard/views.py

```diff
@@ -824,56 +824,58 @@ def apitos(request):
     if access_token and is_github_token_valid(access_token):
         helper_handle_access_token(request, access_token)
 
-    basic_tools = Tool.objects.filter(category='BASIC')
-    advanced_tools = Tool.objects.filter(category='ADVANCED')
-    community_tools = Tool.objects.filter(category='COMMUNITY')
-    to_build_toos = Tool.objects.filter(category='TOOLS_TO_BUILD')
-    alpha_tools = Tool.objects.filter(category='ALPHA')
-    coming_soon_tools = Tool.objects.filter(category='COMING_SOON')
-    for_fun_tools = Tool.objects.filter(category='FOR_FUN')
+    tools = Tool.objects.prefetch_related('votes').all()
+
     actors = [{
         "title": _("Basics"),
         "description": _("Accelerate your dev workflow with Gitcoin\'s incentivization tools."),
-        "tools": basic_tools
-      }, {
-          "title": _("Advanced"),
-          "description": _("Take your OSS game to the next level!"),
-          "tools": advanced_tools
-      }, {
-          "title": _("Community"),
-          "description": _("Friendship, mentorship, and community are all part of the process."),
-          "tools": community_tools
-       }, {
-          "title": _("Tools to BUIDL Gitcoin"),
-          "description": _("Gitcoin is built using Gitcoin.  Purdy cool, huh? "),
-          "tools": to_build_toos
-       }, {
-          "title": _("Tools in Alpha"),
-          "description": _("These fresh new tools are looking for someone to test ride them!"),
-          "tools": alpha_tools
-       }, {
-           "title": _("Tools Coming Soon"),
-           "description": _("These tools will be ready soon.  They'll get here sooner if you help BUIDL them :)"),
-           "tools": coming_soon_tools
-       }, {
-           "title": _("Just for Fun"),
-           "description": _("Some tools that the community built *just because* they should exist."),
-           "tools": for_fun_tools
-        }]
+        "tools": tools.exclude(category__ne=Tool.CAT_BASIC)
+    }, {
+        "title": _("Advanced"),
+        "description": _("Take your OSS game to the next level!"),
+        "tools": tools.exclude(category__ne=Tool.CAT_ADVANCED)
+    }, {
+        "title": _("Community"),
+        "description": _("Friendship, mentorship, and community are all part of the process."),
+        "tools": tools.exclude(category__ne=Tool.CAT_COMMUNITY)
+    }, {
+        "title": _("Tools to BUIDL Gitcoin"),
+        "description": _("Gitcoin is built using Gitcoin.  Purdy cool, huh? "),
+        "tools": tools.exclude(category__ne=Tool.CAT_BUILD)
+    }, {
+        "title": _("Tools in Alpha"),
+        "description": _("These fresh new tools are looking for someone to test ride them!"),
+        "tools": tools.exclude(category__ne=Tool.CAT_ALPHA)
+    }, {
+        "title": _("Tools Coming Soon"),
+        "description": _("These tools will be ready soon.  They'll get here sooner if you help BUIDL them :)"),
+        "tools": tools.exclude(category__ne=Tool.CAT_COMING_SOON)
```

Comment:
wouldnt this be `tools.filter` not `tools.exclude`?

Reply:
oh i guess the _ne takes care of that... but feels like a double negative to me

Reply:
@owocki `QuerySet.exclude` with `ne` reduces the query results from `.all()` - I'll switch it back to filter, though.